### ISSUE TITLE
Operations never resolve when using the `idb` adapter

### ISSUE THREAD

**Issue by TillaTheHun0** [aaronhuggins/pouchdb_deno#6]
Firstly, really awesome work here. I'm so excited to be able to use PouchDB in Deno!

Something I came across, maybe I am missing something, it seems that operations hang when using the `idb` adapter. That is to say, the Promise never resolves.

Reproduce:
```js
import PouchDB from "https://deno.land/x/pouchdb_deno@v1.0.0-PouchDB+7.2.2/modules/pouchdb/mod.ts";

const db = new PouchDB('foobar', { adapter: 'idb' })

console.log(db)

// This Promise seems to never resolve
console.log(await db.post({
  name: 'foo'
}))

console.log(await db.find({ selector: { name: 'foo' }}))
```

I am able to use the `memory` adapter and things seem to work, so seems like something with `idb` polyfill?

Thanks again.

**Comment by aaronhuggins** [comment]
Hey! I had a quick minute, so a couple of things:

- Upgrade to latest version for some IndexedDB fixes I released a few weeks ago: [v1.1.1-PouchDB+7.2.2](https://github.com/aaronhuggins/pouchdb_deno/releases/tag/v1.1.1-PouchDB%2B7.2.2)
- Check to see if your DB on-disk is corrupted; I was able to repro corruption by manually deleting the `D__pouch_foobar.sqlite` and leaving the IndexedDB index file `__sysdb__.sqlite`, which caused your reported behavior on v1.1.1
- Try wiping out your sqlite DBs completely and running again, if an option

`v1.0.0` had an issue which could cause db corruption on script exit, so I'm not really surprised.

Otherwise, a clean fresh DB on latest version I cannot repro! :smiley:
